db/migrate/20250719140801_change_ticket_log_action_type_enum.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class ChangeTicketLogActionTypeEnum < ActiveRecord::Migration[7.2]
+  def up
+    execute <<-SQL.squish
+      UPDATE ticket_logs
+      SET action_type = CASE action_type
+                        WHEN 'created' THEN 'create_ticket'
+                        WHEN 'status_updated' THEN 'update_status'
+                        ELSE action_type
+                        END;
+    SQL
```

Comment:
You can write this in Rails language:
```ruby
TicketLog.where(action_type: 'created').update_all(action_type: 'create_ticket')
```

Reply:
t would require one line (ie one `UPDATE` statement) per type, instead of a compact `CASE`. But it's more maintainable if we ever need to revisit or adjust this migration.

Reply:
But I am not sure how Rails will react to the fact that you're changing enum values that "don't exist" anymore (because you're hard-replacing them in the model code above). Have you tried the AR approach before?

Reply:
Cool! I think this has to do with the fact that `update_all` does not run any form of validations. It's basically a "raw translation" of SQL `UPDATE` statement. Only `update` and `update!` that are called on an individual model go through validations.